terraform/app/terraform.tf

```diff
@@ -48,54 +43,56 @@ terraform {
 
 
 module cloudfront {
-  source = "./modules/cloudfront"
-
-  environment                   = terraform.workspace
+  source                        = "./modules/cloudfront"
+  for_each                      = var.cloudfront_distributions
```

Comment:
The name of the variable shadows the `for_each` reserved keyword. It may be confusing for the reader.
Can we change it to something like `distribution_list`?

Reply:
Variable is now `distribution_list`